### ISSUE TITLE
Cross-package tests

### ISSUE THREAD

**Issue by henryiii** [scikit-hep/scikit-hep#398]
There should be some tests that make sure the various metapackage items work together.

**Comment by eduardo-rodrigues** [comment]
Indeed. So far I've been doing this by hand, for example installing everything in a docker container. But one should be open to any other and better ideas :-).

**Comment by eduardo-rodrigues** [comment]
Indeed such a check would be useful. For example, Python 3.13 has been added but while checking for suppport across the dependencies, for a release https://github.com/scikit-hep/scikit-hep/pull/401, I noticed that hepstats doesn't yet support 3.13.

FYI @jonas-eschle.

**Comment by eduardo-rodrigues** [comment]
Cool, ping me here or on https://github.com/scikit-hep/scikit-hep/pull/401 when sorted :+1:.

**Comment by hjaat** [comment]
"Hi! I'd like to work on this issue.
I'm a student interested in HEP
and would like to contribute."

**Comment by eduardo-rodrigues** [comment]
Hello @hjaat. Thank you for the interest. Sure, you're very welcome to contribute to this. We appreciate community contributions a lot.

I see you already got started with a PR ... Let's talk there ...

For reference: I still do not know the best way to "relate" this task to what is presently in the repo https://github.com/scikit-hep/integration-tests and what could be further enhanced there. In any case, some testing here makes sense.

FYI @henryiii.
